### ISSUE TITLE
BootstrapIcons partially broken?

### ISSUE THREAD

**Issue by jstueckrath** [kordamp/ikonli#134]
I am working on a JavaFX application and included the ikonli-bootstrapicons-pack, but noticed that some symbols are not as they should be. There are several symbols that should not be filled but are and there are filled icons missing e.g. arrows within. I noticed the issue in my own application, the online cheat sheat and the ikonli-browser.

See for instance the symbols 31, 34, 35, 36 in this cheat sheat:
https://kordamp.org/ikonli/cheat-sheet-bootstrapicons.html

They are all missing the inner arrows and some should not be filled (I compared with https://icons.getbootstrap.com/).

The ikonli-browser looks like this:
![ikonli-browser](https://user-images.githubusercontent.com/38353563/105637499-267be780-5e6e-11eb-84ea-34a8ed595ff1.png)

Am I missing something? Is there a problem with the icon pack or the used Bootstrap version?

**Comment by aalmiray** [comment]
The icon order between Ikonli and https://icons.getbootstrap.com is different due to 2 reasons:
- Ikonli sorts icons by name
- Ikonli `12.1.0` ships with bootstrapicons `1.2.1` while the icon page (currently) shows `1.3.0`.

Here's how ikonli-browser displays icons on my machine. There are no missing icons that I can see

<img width="806" alt="bootstrapicons" src="https://user-images.githubusercontent.com/13969/105643758-14ac3b80-5e92-11eb-9547-0045ffd3c98a.png">

```sh
------------------------------------------------------------
Gradle 6.3
------------------------------------------------------------

Build time:   2020-03-24 19:52:07 UTC
Revision:     bacd40b727b0130eeac8855ae3f9fd9a0b207c60

Kotlin:       1.3.70
Groovy:       2.5.10
Ant:          Apache Ant(TM) version 1.10.7 compiled on September 1 2019
JVM:          11.0.9 (Azul Systems, Inc. 11.0.9+11-LTS)
OS:           Mac OS X 10.15.7 x86_64
```

**Comment by jstueckrath** [comment]
Thank you for your quick reply. I found out that there actually is an issue with bootstrapicons, but it only affects Windows (which seems to have a problem with inconsistent fill rules) and they already fixed it in version 1.2.2:
https://blog.getbootstrap.com/2020/12/23/bootstrap-icons-1-2-2/

I downloaded the font directly from them and you can see the difference below (left is 1.2.1, right is 1.2.2).
<img src="https://user-images.githubusercontent.com/38353563/105698629-6a223000-5f06-11eb-9ec0-6c32e5fe4614.PNG" width="200"/> <img src="https://user-images.githubusercontent.com/38353563/105698635-6bebf380-5f06-11eb-971d-8aa3be64940d.PNG" width="200"/>

Thus, would you please consider upgrading to a newer bootstrapicons version?

**Comment by aalmiray** [comment]
Of course, not a problem.
